assets/helpers/contributions.js

```diff
@@ -21,43 +26,45 @@ export type Amounts = {
   oneOff: Amount,
 };
 
+export type validatedContrib = {
+  amount: number,
+  error: ContribError,
+};
+
 
 // ----- Setup ----- //
 
-const limits = {
+const LIMITS = {
```

Comment:
Would it make more sense to split it product.parameter instead of parameter.product?
```
const config = {
  RECURRING: {
    min: 5,
    max: 2000,
    default: 10,
  }
  ONE-OFF: {
    min: 1,
    max: 2000,
    default: 50,
  }
}
```